app/views/users/edit.html.haml

```diff
@@ -14,6 +14,18 @@
         = link_to 'Change your avatar here', 'https://gravatar.com'
         = f.input :affiliation, as: :string,
           hint: 'This could be a company, a user group, or nothing at all.'
+        = f.input :website_url, label: 'Personal Website'
+        = f.inputs :name => 'Social Media' do
+          = f.input :linkedin
```

Comment:
We use html5 so you could use input types:
http://www.w3schools.com/html/html_form_input_types.asp